lib/repl.js

```diff
@@ -429,7 +430,9 @@ function REPLServer(prompt,
       const { pathToFileURL } = require('url');
       // Adding `/repl` prevents dynamic imports from loading relative
       // to the parent of `process.cwd()`.
-      parentURL = pathToFileURL(path.join(process.cwd(), 'repl')).href;
+      parentURL = pathToFileURL(
+        ArrayPrototypeJoin(path, process.cwd(), 'repl')
+      ).href;
```

Comment:
`path.join` is from the Node.js module, not the array method

Reply:
whoops, thanks, will fix :-)